docker-compose.yml

```diff
@@ -124,6 +124,7 @@ services:
       - CONTACT_US_EMAIL_ADDRESS
       - FF_TFM_PAYMENT_RECONCILIATION_ENABLED
       - FF_TFM_FACILITY_END_DATE_ENABLED
+      - FF_TFM_DEAL_CANCELLATION
```

Comment:
I can also pre-emptively add this to tfm-api (& central-api?) if we expect we'll be using them in either place?

Reply:
That's a really good point, I think either way is fine -- it might be easier to do this as soon as we create a new route in tfm-ui (or anywhere else), as we want to get common middleware in place for this...

Reply:
Sure, I'll leave it for now